tests/mod-resolver/test-submodule-issue-5119/Cargo.toml

```diff
@@ -4,5 +4,4 @@ version = "0.1.0"
 edition = "2018"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
```

Comment:
Not talked in style guide, but maybe unexpected

Reply:
Can we hold off on modifying existing Cargo.toml files until a follow up PR